geocoder_tester/world/france/test_synonymes.csv

```diff
@@ -0,0 +1,7 @@
+query,lon,lat,limit,expected_name,expected_housenumber,expected_street,expected_city
+2 rue du 10e niort,,,4,,2,Rue du Dixième,Niort
+4 rue 11 novembre Frénouville,,,4,,4,Rue du Onze Novembre,Frénouville
```

Comment:
when I search for this one in all the geocoders I find `rue 11 novembre`

Reply:
the expected answer is this one : https://www.openstreetmap.org/way/333816885